.github/workflows/_test_build_fake_prerelease.yaml

```diff
@@ -0,0 +1,84 @@
+name: Test Build *Nix
+
+on:
+  workflow_call:
+    inputs:
+      integration:
+        required: true
+        type: string
+    secrets:
+      OHAI_PFX_CERTIFICATE_BASE64:
+        required: true
+      OHAI_PFX_PASSPHRASE:
+        required: true
+
+jobs:
+  test-build-nix:
+    name: Test binary compilation for all platforms:arch
+    runs-on: ubuntu-latest
+    env:
+      INTEGRATION: ${{ inputs.integration }}
+      REPO_FULL_NAME: ${{ github.event.repository.full_name }}
+      TAG: "v0.0.0"
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          git tag "$TAG"
+          if [ -z "$GPG_PASSPHRASE" ]; then
+            echo NO_SIGN=true >> $GITHUB_ENV
+          fi
+      - name: Build all platforms:arch
+        run: make ci/fake-prerelease
+      - name: Upload artifacts for next job
+        uses: actions/upload-artifact@v3
+        with:
+          name: windows-packages
```

Comment:
the job is called test-build-nix, but we are uploading also windows packages.
Maybe rebuilding them is faster then having a need condition and we remove a dependency

Reply:
I tried, but I had some issues building it for Windows since this [Dockerfile](https://github.com/newrelic/nri-apache/blob/master/build/Dockerfile) we use when running `make ci/fake-prerelease` is not compatible with Windows.